docuware-connector/src/com/axonivy/connector/docuware/connector/utils/JsonUtils.java

```diff
@@ -0,0 +1,138 @@
+package com.axonivy.connector.docuware.connector.utils;
+
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+
+import com.axonivy.connector.docuware.connector.DocuWareProperties;
+import com.axonivy.connector.docuware.connector.DocuWarePropertiesUpdate;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.util.StringUtil;
+
+/**
+ * Provides utilities related to json serialization like building an ObjectMapper with all required options and features.
+ *
+ * @author jpl
+ *
+ */
+public class JsonUtils {
```

Comment:
heavy hammer that invents lot of code already present in the JAX-RS stack.
we only need it read/write this properites file right?

Reply:
yes, i removed all unused methods